frontend/views/containers/chatroom/image-viewer/PreviewImageArea.vue

```diff
@@ -0,0 +1,457 @@
+<template lang="pug">
+.c-image-view-area(
+  @wheel.prevent.stop='wheelEventHandler'
+)
+  img.c-preview-image(ref='previewImg'
+    :class='{ "is-movable": isImageMovable }'
+    :src='imgSrc'
+    :width='config.imgData.naturalWidth'
+    :height='config.imgData.naturalHeight'
+    :style='previewImgStyles'
+    :alt='name'
```

Comment:
Is "name" a good `alt` for this? Seems a bit unclear to me.

Maybe "Image preview"? And probably should be wrapped in an `L` function for localization.

Reply:
@taoeffect OK, will use `L('Image preview')` then.